pre_commit/checkers/ruby_version_checker.rb

```diff
@@ -12,12 +12,14 @@ def errors?
   end
 
   def examine_code
-    file_is_forbidden? ? [%{Edit to #{@file} - you probably didn't mean to do commit that.}] : []
+    mess = []
+    @files.each do |file|
+      base = File.basename(file)
+      if base == '.ruby-version' || base == '.rbenv-version'
```

Comment:
Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.